equal.js

```diff
@@ -0,0 +1,17 @@
+//1) what are our parameters?
+//-- an array of numbers
+
+// 2) what are we returning?
+//-- true or false
+
+const trueequal = numsArray => {
+    var equal = 0
+    
```

Comment:
This was just looking for a basic function that would result in strict equality.